CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `load_model` function ([#213](https://github.com/MobileTeleSystems/RecTools/pull/213))
 - `model_from_config` function ([#214](https://github.com/MobileTeleSystems/RecTools/pull/214))
 - `get_cat_features` method to `SparseFeatures` ([#221](https://github.com/MobileTeleSystems/RecTools/pull/221))
+- Support `fit_partial()` for LightFM ([#223](https://github.com/MobileTeleSystems/RecTools/pull/223))
```

Comment:
Let's pull `main` branch and we are ready for merging.

Reply:
Resolve the conflict, thanks!